src/main/java/healthy/lifestyle/backend/mentals/service/MentalServiceImpl.java

```diff
@@ -0,0 +1,76 @@
+package healthy.lifestyle.backend.mentals.service;
+
+import healthy.lifestyle.backend.exception.ApiException;
+import healthy.lifestyle.backend.exception.ErrorMessage;
+import healthy.lifestyle.backend.mentals.dto.MentalResponseDto;
+import healthy.lifestyle.backend.mentals.model.Mental;
+import healthy.lifestyle.backend.mentals.repository.MentalRepository;
+import healthy.lifestyle.backend.mentals.repository.MentalTypeRepository;
+import healthy.lifestyle.backend.users.model.User;
+import healthy.lifestyle.backend.users.service.UserService;
+import healthy.lifestyle.backend.workout.dto.HttpRefResponseDto;
+import healthy.lifestyle.backend.workout.repository.HttpRefRepository;
+import java.util.Comparator;
+import java.util.List;
+import org.modelmapper.ModelMapper;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+public class MentalServiceImpl implements MentalService {
+
+    private final MentalRepository mentalRepository;
+
+    private final HttpRefRepository httpRefRepository;
+
+    private final UserService userService;
+
+    private final MentalTypeRepository mentalTypeRepository;
+
+    private final ModelMapper modelMapper;
+
+    public MentalServiceImpl(
+            MentalRepository mentalRepository,
+            HttpRefRepository httpRefRepository,
+            UserService userService,
+            MentalTypeRepository mentalTypeRepository,
+            ModelMapper modelMapper) {
+        this.mentalRepository = mentalRepository;
+        this.httpRefRepository = httpRefRepository;
+        this.userService = userService;
+        this.mentalTypeRepository = mentalTypeRepository;
+        this.modelMapper = modelMapper;
+    }
+
+    @Override
+    @Transactional
```

Comment:
Since we don't change anything in the database, we can omit `@Transactional`. 
We use it to ensure that if we have many transactions then all these transactions will be treated as a single unit to preserve the database consistency. If one transaction is failed then all others are not committed to the database.